go.mod

```diff
@@ -1,6 +1,7 @@
 module kubevirt.io/containerdisks
 
 go 1.22.9
+toolchain go1.23.4
```

Comment:
Better to update the version instead.

Reply:
@jcanocan can we just do that in a follow up?

Reply:
Yep, I'm working on it.